blocks/identity-block/features/header-account-action/default.jsx

```diff
@@ -66,17 +66,54 @@ const HeaderAccountAction = ({ customFields }) => {
 
   if (user && !error) {
     return (
-      <div className="xpmedia-subs-header">
-        <p>{user.displayName}</p>
-        <button type="button" onClick={handleLogout} className="xpmedia-subs-header--button">Log Out</button>
+      <div className="xpmedia-subs-header--header">
+        <div className="xpmedia-subs-header--desktop-header">
+          <Button
+            aria-expanded={isAccountMenuOpen}
+            as="button"
+            buttonSize={BUTTON_SIZES.SMALL}
+            buttonStyle={BUTTON_STYLES.WHITE_BACKGROUND_FILLED}
+            buttonType={BUTTON_TYPES.LABEL_AND_TWO_ICONS}
+            iconType="user"
+            secondaryIconType={isAccountMenuOpen ? 'chevron-up' : 'chevron-down'}
+            onClick={() => setAccountMenu(!isAccountMenuOpen)}
+            text={phrases.t('identity-block.account')}
+            type="button"
+          />
+        </div>
+        <div className="xpmedia-subs-header--mobile-header">
+          <Button
+            aria-expanded={isAccountMenuOpen}
+            as="button"
+            buttonSize={BUTTON_SIZES.SMALL}
+            buttonStyle={BUTTON_STYLES.WHITE_BACKGROUND_FILLED}
+            buttonType={BUTTON_TYPES.ICON_ONLY}
+            leftIconType="user"
+            onClick={() => setAccountMenu(!isAccountMenuOpen)}
+            text={phrases.t('identity-block.login-options')}
+            type="button"
+          />
+        </div>
+        {isAccountMenuOpen && (
+          <ul className="xpmedia-subs-header-dropdown--open">
+            <DropDownLinkListItem
+              href="/"
+              text={phrases.t('identity-block.manage-account')}
+            />
+            <DropDownLinkListItem
+              href="/"
```

Comment:
Is logout to take you to the root? Also, this link will not log out the user - just navigate them to the `href` - The `href` value should be a custom field value

Reply:
updated